<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Mer::Internal::MerVirtualMachineSettingsWidget</class>
 <widget class="QWidget" name="Mer::Internal::MerVirtualMachineSettingsWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>146</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <property name="horizontalSpacing">
    <number>6</number>
   </property>
   <property name="verticalSpacing">
    <number>6</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QLabel" name="memoryLabel">
     <property name="text">
      <string>Memory:</string>
     </property>
     <property name="buddy">
      <cstring>memorySpinBox</cstring>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QSpinBox" name="memorySpinBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="suffix">
      <string> MB</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="cpuLabel">
     <property name="text">
      <string>Processor(s):</string>
     </property>
     <property name="buddy">
      <cstring>cpuCountSpinBox</cstring>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QSpinBox" name="cpuCountSpinBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="vdiCapacityLabel">
     <property name="text">
      <string>Vdi capacity:</string>
     </property>
     <property name="buddy">
      <cstring>vdiCapacityGbSpinBox</cstring>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QDoubleSpinBox" name="vdiCapacityGbSpinBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="suffix">
      <string> GB</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="vdiSizeOnDiskLabel">
     <property name="text">
      <string>Size on disk:</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QLabel" name="vdiSizeOnDiskValueLabel">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="vdiPathLabel">
     <property name="text">
      <string>Path:</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QLabel" name="vdiPathLabelEdit">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
     <property name="textInteractionFlags">
      <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByKeyboard|Qt::TextSelectableByMouse</set>
     </property>
    </widget>
   </item>
  </layout>
  <zorder>memoryLabel</zorder>
  <zorder>cpuLabel</zorder>
  <zorder>vdiCapacityLabel</zorder>
  <zorder>vdiSizeOnDiskLabel</zorder>
  <zorder>vdiSizeOnDiskValueLabel</zorder>
  <zorder>vdiPathLabel</zorder>
  <zorder>vdiPathLabelEdit</zorder>
  <zorder>memorySpinBox</zorder>
  <zorder>cpuCountSpinBox</zorder>
  <zorder>vdiCapacityGbSpinBox</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
